src/validate-text.ts

```diff
@@ -115,15 +115,15 @@ async function validateText(textToBeValidated: string, options?: ValidateTextOpt
 	cssValidationResponse.errors?.forEach((error) => {
 		result.errors.push({
 			line: error.line,
-			message: error.message,
+			message: error.message.replace(' : ', '').trim(),
 		});
 	});
 
 	if ('warnings' in result) {
 		cssValidationResponse.warnings?.forEach((warning) => {
 			result.warnings.push({
 				line: warning.line,
-				message: warning.message,
+				message: warning.message.replace(' : ', '').trim(),
```

Comment:
See comment on the line above:

```suggestion
				message: warning.message.replace(/[ :]+$/, '').trim(),
```